config/locales/server.fa_IR.yml

```diff
@@ -6,9 +6,7 @@
 
 fa_IR:
   topic:
-    qa:
-      errors:
-        qa_not_enabled: "پرسش و پاسخ فعال نیست."
```

Comment:
Is the removal of some keys intentional?

Reply:
Yeah, I honestly couldn't decide if they should stay or go away. 

- stay: the translations are inaccurate
- go away: they don't show up at all

I'll go with stay, and add them back